### PR TITLE
Cleanup jboss module

### LINKED ISSUE #18600: modules.jboss7 pylint issues
```
************* Module salt.modules.jboss7
salt/modules/jboss7.py:74: [W0622(redefined-builtin), reload] Redefining built-in 'reload'
salt/modules/jboss7.py:31: [W0613(unused-argument), status] Unused argument 'timeout'
salt/modules/jboss7.py:149: [W0622(redefined-builtin), __format_value] Redefining built-in 'type'
```

### PR DESCRIPTION
**Pull request by cachedout**
Closes #18600 
Also adds beryllium release notes

### CODE REVIEW

File: doc/topics/releases/beryllium.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+=======================================
+Salt Release Notes - Codename Beryllium
+=======================================
+
+JBoss 7 State
+=============
+
+- Renamed ``reload`` function to ``reload_jboss``
```
Comment (cvrebert) [inline doc/topics/releases/beryllium.rst]: You could use `__func_alias__` to keep the externally-visible name the same

**Comment by thatch45** [comment]
@cachedout please change to use __func_alias__, then it can stay reload and we don't need to make the api change

**Comment by cachedout** [comment]
How did I not know that was a thing?! Crazy. I learn something new every day I guess. Will fix.

**Comment by thatch45** [comment]
it is a great thing :)

**Pull request merged**